boilerplate/test/setup.ts

```diff
@@ -34,6 +34,11 @@ jest.mock("i18n-js", () => ({
   },
 }))
 
+jest.mock("expo-localization", () => ({
+  ...jest.requireActual("expo-localization"),
+  getLocales: () => [{ languageTag: "en-US", textDirection: "ltr" }],
```

Comment:
although `expo-localization` returns `getLocales`, it returns an empty array. setting a default here